tests/functional/lib/helpers.js

```diff
@@ -448,6 +448,29 @@ define([
   });
 
   /**
+   * Get the email headers
+   *
+   * @param {string} user - username or email address
+   * @param {number} index - email index.
+   * @param {object} [options]
+   *   @param {number} [options.maxAttempts] - number of email fetch attempts
+   *   to make. Defaults to 10.
+   * @returns {promise} resolves with the email headers if email is found.
+   */
+  const getEmailHeaders = thenify(function(user, index, options) {
+    if (/@/.test(user)) {
+      user = TestHelpers.emailToUser(user);
+    }
+
+    // restmail takes an index that is 1 based instead of 0 based.
```

Comment:
I was a bit confused by this comment so went to look at the code. Would it be more accurate to say that restmail takes a length, hence adding 1 to the index?